### ISSUE TITLE
Issues with using startCassandra in IntegrationTest

### ISSUE THREAD

**Issue by aaruna** [hochgi/sbt-cassandra-plugin#5]
Instead of adding startCassandra dependency to tests `test in Test <<= (test in Test).dependsOn(startCassandra)` , I am trying to add the same to IntegrationTest

`test in IntegrationTest <<= (test in IntegrationTest).dependsOn(startCassandra)`.

Everything works fine except the following two issues

* `startCassandra` is run even before `Compile in IntegrationTest` is run.
* `stopCassandra` is not run after tests are executed.

From what I see in `com.github.hochgi.sbt.cassandra.CassandraPlugin`, I see that the plugin is adding this.

L151
`startCassandra <<= startCassandra.dependsOn(compile in Test)`

L157
```
testOptions in Test <+=.....
```

So the workaround for me currently is to add these in my build.

**Comment by hochgi** [comment]
Yes. It is kind of hacky. I'm planning a rewrite of the plugin for awhile, So it would work for every Configuration, and also promote the plugin to be AutoPlugin. Hope to get to it soon. (will happily merge a PR)
Thanks.
